hello-world/hello-world-theme/todo.hubl.html

```diff
@@ -7,8 +7,11 @@
 
 {% block body %}
 
-  {% module "todo_js_module"
```

Comment:
@timmfin any opinion if we should keep this AND the dnd_module example, maybe devs would assume they could only use it with`dnd_module` 🤷

Reply:
No strong opinion... though we could keep `{% module "todo_js_module" .. %}` around but commented so that example is obvious.